packages/styles/src/flags/README.md

```diff
@@ -0,0 +1,46 @@
+# Flag
+
+Flag displays contextual information about events on the page.
+
+## Default
+
+```html
+<div class="as-flag">
+  <div class="as-flag__icon">
+    <svg width="12px" height="13px" viewBox="0 0 16 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
IMO using Inline svgs in the examples adds a lot of noise

Reply:
Yep, I know, we can change it later when we have the `<as-icon />` component

Reply:
icons are ready now, you can use the svg version here ^^

https://carto-airship.netlify.com/catalog/#/icons/icons